relay_test.go

```diff
@@ -89,17 +89,23 @@ func TestRelay(t *testing.T) {
 	})
 }
 
-func DisabledTestRelayHandlesCrashedPeers(t *testing.T) {
-	withRelayedEcho(t, func(_, server, client *Channel, ts *testutils.TestServer) {
-		ctx, cancel := NewContext(time.Second)
+func TestRelayHandlesClosedPeers(t *testing.T) {
+	opts := serviceNameOpts("test").SetRelayOnly().
```

Comment:
Is this an accepted formatting choice? Feel like a chain, when it is multi line, should probably be:

```
Thing().
ThingTwo().
ThingThree()
```